packages/ember/embroider/vitest.config.ts

```diff
@@ -3,7 +3,7 @@ import { defineConfig } from 'vite';
 
 export default defineConfig({
   test: {
-    testTimeout: 30_000,
+    testTimeout: 60_000,
```

Comment:
I had to bump these because they ran in about 35 seconds for me locally, not sure how fast it is in CI.

Reply:
this is fine -- the tests are slow, but of high value

Reply:
I rebased on main, and now I'm even seeing timeout issues (on mac) with this value 😄.

Reply:
Looks like snapshots need updated, since node updated or something

Reply:
It seems these are failing on main already. Not sure if this PR should resolve those?

Reply:
That's fair